huxley/www/js/lib/ServerAPI.js

```diff
@@ -15,6 +15,10 @@ var ServerAPI = {
     return _post('/api/delegates', {name, email, school});
   },
 
+  createCommitteeFeedback(feedback) {
+    return _post(`/api/committee_feedback/0`, feedback);
```

Comment:
There should be a way to post without needing an id value here. It will need a change at the api level though

Reply:
Like we discussed, we'll add a fixed url to handle single post requests. I think it's the cleanest way to do it.